javascript/javascript2/week2/doublingnumber.js

```diff
@@ -0,0 +1,4 @@
+let numbers = [1, 2, 3, 4];
+console.log("original Array "+numbers);
+let doubledEvenNumbers = numbers.filter(number => number% 2 !== 0).map(number => number* 2);
```

Comment:
Add some extra spaces to make the code easier to read.
```suggestion
let doubledEvenNumbers = numbers.filter(number => number % 2 !== 0).map(number => number * 2);
```